walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/constraints/ClaimRegexConstraint.kt

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.walletlibrary.requests.requirements.constraints
+
+import com.jayway.jsonpath.Configuration
+import com.jayway.jsonpath.JsonPath
+import com.jayway.jsonpath.Option
+import com.microsoft.did.sdk.credential.models.VerifiableCredentialContent
+import com.microsoft.did.sdk.util.log.SdkLog
+import com.microsoft.walletlibrary.util.VerifiedIdIssuerIsNotRequestedException
+import com.microsoft.walletlibrary.verifiedid.VerifiableCredential
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import kotlinx.serialization.json.Json
+import java.util.regex.Pattern
+
+class ClaimRegexConstraint(internal val path: List<String>, internal val pattern: String) :
+    VerifiedIdConstraint {
+    override fun doesMatch(verifiedId: VerifiedId): Boolean {
+        if (verifiedId !is VerifiableCredential)
+            return false
+        val vccJsonString =
```

Comment:
nit: naming here. what is vcc?

Reply:
that is a copy paste issue. I will change it.